README.md

```diff
@@ -291,6 +291,46 @@ You must define general parameters like the base_url and the website title which
     website.title = My Company Intranet
     website.base_url = http://intranet.mycompany.com:8080
 
+#### LDAP ####
+
+To use LDAP authentication, set ``auth_type`` parameter to "ldap":
+
+    auth_type = ldap
+
+Then add LDAP parameters
+
+    # LDAP server address
+    ldap_url = ldap://localhost:389
+
+    # Base dn to make queries
+    ldap_base_dn = dc=directory,dc=fsf,dc=org
+
+    # Bind dn to identify the search
+    ldap_bind_dn = cn=admin,dc=directory,dc=fsf,dc=org
+
+    # The bind password
+    ldap_bind_pass = toor
+
+    # Attribute name of user record who contain user login (email)
+    ldap_ldap_naming_attribute = uid
+
+    # Matching between ldap attribute and ldap user field (ldap_attr1=user_field1,ldap_attr2=user_field2,...)
+    ldap_user_attributes = mail=email
+
+    # TLS usage to communicate with your LDAP server
+    ldap_tls = False
+
+    # If True, LDAP own tracim group managment (not available for now!)
+    ldap_group_enabled = False
+
+You may need an administrator account to manage Tracim. Use the following command (from ``/install/dir/of/tracim/tracim``):
+
+```
+gearbox user create -l admin-email@domain.com -p new_password -g managers -g administrators

```

Comment:
it's ``user create``: ``setup.py`` commands is ``user_create`` but ``_`` are replaced by spaces.